website_docs/using_instrumentation_libraries.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Using instrumentation libraries
+weight: 3
+---
+
+Go does not support truly automatic instrumentation like other languages today. Instead, you'll need to depend on [instrumentation libraries](https://opentelemetry.io/docs/reference/specification/glossary/#instrumentation-library) that generate telemetry data for a particular instrumented library. For example, the instrumentation library for `net/hhtp` will automatically create spans that track inbound and outbound requests once you configure it in your code
+
+## Setup
+
+Each instrumentation library is a package. In general, this means you need to `go get` the appropriate package:
+
+```console
+go get go.opentelemetry.io/contrib/instrumentation/{import-path}/otel{package-name}
+```
+
+And then configure it in your code based on what the library requires to be activated.
+
+## Example with `net/http`
+
+As an example, here's how you can set up automatic instrumentation for inbound HTTP requests for `net/http`:
+
+First, get the `net/http` instrumentation library:
+
+```console
+go get go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp
+```
+
+Next, use the library to wrap an HTTP handler in your code:
+
+```go
+module MyApp
+
+// Import the package
+import (
+  // ...
+  "time"
+  "net/http"
+  "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+  // ...
+)
+
+// module-level tracer, configured in your code somewherex
+const tracer
+
+// ...
+
+// Have a function that does some work,
+// which will connect with automatic instruemtnation later in the code
+func sleepy(ctx context.Context) {
+    _, span := tracer.Start(ctx, "sleep")
+    defer span.End()
+
+    sleepTime := 1 * time.Second
+    time.Sleep(sleepTime)
+
+    span.SetAttributes(attribute.Int("sleep.duration", sleepTime))
+}
+
+// Have some http handler function to instrument
+func httpHandler(w http.ResponseWriter, r *http.Request) {
+    fmt.Fprintf(w, "Hello, World! I am instrumented autoamtically!")
+
+    ctx := r.Context()
+
+    // Do some work
+    sleepy(ctx)
+}
+
+//...
+
+// Instantiate and wrap the httpHandler function that you just defined
+handler := http.HandlerFunc(httpHandler)
+wrappedHandler := otelhttp.NewHandler(handler, "hello-instrumented")
+http.Handle("/hello-instrumented", wrappedHandler)
+
+// And start the HTTP server
+log.Fatal(http.ListenAndServe(":3030", nil))
```

Comment:
Yes, thank you! This is great. I had some compilable code I was considering using, but it was kinda overwhelming since it covered setting up an exporter, tracer provider, setting context propagation, etc. This is far better for this document.